code/deposit-calculator/src/test/java/ru/unn/agile/DepositCalculator/Model/CalculatorTest.java

```diff
@@ -0,0 +1,256 @@
+package ru.unn.agile.DepositCalculator.Model;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class CalculatorTest {
+
+    private final double delta = 0.01;
+
+    @Test
+    public void caclYearDepositWithoutCapitalization() {
+        Calculator calc = new Calculator();
+        double depositRes = calc.setStartSum(100)
```

Comment:
Please create a method for calculation to make code more compact

Reply:
I prefer keep in as-is. Method won't be compact because of amount of arguments.